src/main/java/it/eng/idsa/businesslogic/processor/receiver/websocket/server/FileRecreatorBeanServer.java

```diff
@@ -88,8 +88,7 @@ private String recreateMultipartMessageFromReceivedFrames() {
 			multipartMessage = recreateMultipartMessage(this.fileByteArray);
 			// once bytes are consumed to get message - empty array
 			fileByteArray = new ArrayList<>();
-			logger.info(
-					"Recreated the Multipart message from the received frames: lenght=" + multipartMessage.length());
+			logger.info("Recreated the Multipart message from the received frames: lenght= {}", multipartMessage.length());
 		} catch (IOException e) {
 			// TODO Auto-generated catch block
 			e.printStackTrace();
```

Comment:
Can we log this exception instead printing it like this?

Reply:
Nice catch on that one, I will check the rest of the code for that